database/src/main/java/org/togetherjava/tjbot/db/Database.java

```diff
@@ -46,10 +46,8 @@ public Database(String jdbcUrl) throws SQLException {
         SQLiteDataSource dataSource = new SQLiteDataSource(sqliteConfig);
         dataSource.setUrl(jdbcUrl);
 
-        Flyway flyway = Flyway.configure()
-            .dataSource(dataSource)
-            .locations("/org/togetherjava/tjbot/db")
-            .load();
+        Flyway flyway =
+                Flyway.configure().dataSource(dataSource).locations("classpath:/db/").load();
```

Comment:
is this an intended change? please double check

Reply:
Yea, it is. I talked with I Al Istannen about it.